akka-actor-typed/src/main/scala/akka/actor/typed/javadsl/Routers.scala

```diff
@@ -64,6 +64,18 @@ abstract class GroupRouter[T] extends DeferredBehavior[T] {
    */
   def withRoundRobinRouting(): GroupRouter[T]
 
+  /**
+   * Route messages by using consistent hashing.
+   *
+   * From wikipedia: Consistent hashing is based on mapping each object to a point on a circle
+   * (or equivalently, mapping each object to a real angle). The system maps each available machine
+   * (or other storage bucket) to many pseudo-randomly distributed points on the same circle.
+   */
+  def withConsistentHashingRouting(
+      virtualNodesFactor: Int,
+      mapping: ConsistentHashMapping[T],
+      system: ActorSystem[T]): GroupRouter[T]
```

Comment:
So one `def withConsistentHashRouting(mapping)` that defaults to `virtualNodesFactor` 1 and one that accepts both a factor and the mapping here.

Reply:
That's incosistency (little pun intended) is on me.